apps/examples/wakaama_client/wakaama_client.c

```diff
@@ -394,6 +314,41 @@ int wakaamaclient_main(int argc, char *argv[])
 			printf("Read a resource value\n");
 			return 0;
 
+		case 'f': {
```

Comment:
We need the opt argc check when we provide option with value. For example _wakaamaclient -s 192.168.1.108_ has argc == 3. So after processing -s, we need to parse one more argument. If by mistake user does not give IP address, the program exits with display message for correct use (hence the return 0 at many places.). We can replace these statements by a macro if you prefer.